source/newcommitter.md

```diff
@@ -229,7 +229,7 @@ sent after a positive result from the vote for a new committer.
 
     A. This personal invitation is a chance for you to 
     accept or decline in private.  Either way, please 
-    let us know in reply to the [private@project.apache.org] 
+    let us know in reply to the private@[PROJECT].apache.org 
```

Comment:
this to align with the cc to `private@[PROJECT].apache.org` written on line https://github.com/apache/comdev-site/pull/73/files#diff-f61968335feadda8b65024fba3abbee7ffe36f56ad106968f47927a9b286f290R205